frame/balances/src/weights.rs

```diff
@@ -18,7 +18,7 @@
 //! Autogenerated weights for pallet_balances
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 2.0.0
-//! DATE: 2021-01-06, STEPS: [50, ], REPEAT: 20, LOW RANGE: [], HIGH RANGE: []
+//! DATE: 2021-01-06, STEPS: \[50, \], REPEAT: 20, LOW RANGE: [], HIGH RANGE: []
```

Comment:
where is this change coming from?

do you need to update the weight template?

Reply:
yes we probably need to update the templates,
rust doc complains the content inside `[]` is not a rust type